hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ListIterator.java

```diff
@@ -191,15 +191,15 @@ public static class CacheIter<Value>
     private final String prefixKey;
     private final String startKey;
     private final String tableName;
-
+    private HeapEntry currentKey;
```

Comment:
nit: `currentEntry` and `getCurrentEntry()`

Reply:
Done.